src/Core/RulesEventManager.php

```diff
@@ -60,4 +60,22 @@ public function processDefinition(&$definition, $plugin_id) {
     }
   }
 
+  /**
+   * Returns the base name of a configured event name.
+   *
+   * For a configured event name like node_view--article the base event name
+   * node_view is returned.
+   *
+   * @return string
+   *   The event base name.
+   */
+  static public function rulesGetEventBaseName($event_name) {
```

Comment:
no, statics should be avoided if possible. the manager is instantiated anyway, so do not make it static